content/docs/enterprise/external-data/zenefits.mdx

```diff
@@ -1,44 +1,112 @@
+    command:
+      - zenefits
+      - --zenefits-api-key=YOUR_API_KEY
+      - --time-zone=America/New_York
+      - --address=:8080
+    container_name: zenefits
+    restart: always
+    expose:
+      - 8080
+```
 </TabItem>
-<TabItem value="dockerhub" label="DockerHub">
+<TabItem value="binary" label="Binary">
+
+These instructions assume a local testing environment. Adjust as needed for your deployment environment.
+
+1. Download the latest [release] of the [Pomerium datasource] project and extract it.
 
+1. Change directory and run the binary:
 
+  ```sh
+  cd datasource-v*
+  ./datasource zenefits --zenefits-api-key=$YOUR_API_KEY --time-zone=America/New_York
+  ```
+
+  The output should resemble:
+
+  ```sh
+  {"level":"info","address":"localhost:8080","message":"ready"}
+  ```
 
 </TabItem>
 </Tabs>
 
+## Configure Pomerium Enterprise Console
 
-To invoke the $EXT_DATA_PROVIDER integration from the datasource, run:
+1. In the Pomerium Enterprise Console, navigate to **CONFIGURE** → **External Data** and click **+ ADD EXTERNAL DATA SOURCE**.
 
-```
-$COMMAND
-```
+1. Fill out the following fields:
 
-## Configure Pomerium Enterprise Console
+  | Field       | Content                          | Notes                                                                     |
+  | :---------- | :------------------------------- | :------------------------------------------------------------------------ |
+  | URL         | `http://zenefits:8080/employees` | Adjust the hostname as needed depending on your local test environment.   |
+  | Record type | `pomerium.io/Zenefits`           |                                                                           |
+  | Foreign Key | `user.email`                     | Pomerium uses the users's email to associate IdP and Bamboo user entries. |
+
+1. Define a new policy. The example policy below only allows persons in the list (i.e. not out of office or on vacation) and in `active` status (i.e. not `terminated` or `onboarding`) to pass. 
+
+  <Tabs>
+  <TabItem value="builder" label="Builder">
+
+  ![Example Zenefits Policy in the Builder view](./img/zenefits-policy.png)
+
+  </TabItem>
+  <TabItem value="editor" label="Editor">
 
+  ```yaml showLineNumbers
+  deny:
+    not:
+      - record:
+          field: status
+          is: Active
+          type: pomerium.io/BambooHRAvailable
+  ```
+
+  </TabItem>
+  </Tabs>
+
+You may add a similar policy around `type` field to i.e. differentiate between employees and contractors. 
 
 ## Reference
 
-[Docker Compose]: https://docs.docker.com/compose/
\ No newline at end of file
+The reference Zenefits data source provides the following record details:
+
+- Zenefits `workEmail` field becomes an `id` of each record, intended to be matched with Pomerium `user.email` foreign key.
+- Zenefits `status` and `type` are exposed as example, and you may build policies around the values of those two fields. See [Zenefits People API] for details. 
```

Comment:
This makes it sound like `status` and `type` are only exposed in the example. What are you trying to say here, so I can massage the words?

Reply:
I mean we're just scratching a surface of use cases that are possible for both Bamboo & Zenefits. They have much more information that potentially may be useful for Pomerium policies - i.e. custom fields, employee location, etc.